src/content/docs/how-to-add-playwright-tests.mdx

```diff
@@ -182,7 +182,7 @@ To run tests with Playwright, check the following commands:
   For example:
 
   ```bash
-  npx playwright test landing-page.spec.ts
+  npx playwright test landing.spec.ts
```

Comment:
it is right that this row is example, and yet -
i thinking about people like me, which has no experience of how-to-run-playwright-tests,
they will copy the example row and try to run it, and may get error...
it may be better to place example that also running...